tests/tools/kopiaclient/kopiaclient.go

```diff
@@ -73,6 +75,24 @@ func (kc *KopiaClient) CreateOrConnectRepo(ctx context.Context, repoDir, bucketN
 	return errors.Wrap(err, "unable to open repository")
 }
 
+// SetCacheLimits sets cache size limits to the already connected repository.
+func (kc *KopiaClient) SetCacheLimits(ctx context.Context, repoDir, bucketName string, cacheOpts *content.CachingOptions) error {
+	err := repo.SetCachingOptions(ctx, kc.configPath, cacheOpts)
+	if err != nil {
+		return err
+	}
+
+	cacheOptsObtained, err := repo.GetCachingOptions(ctx, kc.configPath)
+	if err != nil {
+		return err
+	}
+
+	log.Printf("content cache size" + strconv.FormatInt(cacheOptsObtained.ContentCacheSizeLimitBytes, 10))
+	log.Printf("metadata cache size " + strconv.FormatInt(cacheOptsObtained.MetadataCacheSizeLimitBytes, 10))
```

Comment:
- The format string in `log.Printf` is missing a format specifier. However, `log.Print` would work here.
- The `strconv.FormatInt` call is not needed.

```suggestion
	log.Print("content cache size:", cacheOptsObtained.ContentCacheSizeLimitBytes)
	log.Printf("metadata cache size:", cacheOptsObtained.MetadataCacheSizeLimitBytes)
```